File: myonlinefm/src/main/java/com/sec/myonlinefm/abstructObserver/RequestCallBack.java
package com.sec.myonlinefm.abstructObserver;

/**
 * Created by SRC-TJ-MM-BinYang on 2018/4/13.
 */

public interface RequestCallBack<T> {
    void onSuccess(T val);
    void onFail(String errorMessage);
}

File: myonlinefm/src/main/java/com/sec/myonlinefm/abstructObserver/RequestCallBack.kt
package com.sec.myonlinefm.abstructObserver

/**
 * Created by SRC-TJ-MM-BinYang on 2018/4/13.
 */
interface RequestCallBack<T> {
    fun onSuccess(`val`: T?)
    fun onFail(errorMessage: String?)
}